gym-unity/README.md

```diff
@@ -175,3 +180,147 @@ def main():
 if __name__ == '__main__':
     main()
 ```
+
+## Run Google Dopamine Algorithms
+
+Google provides an implementation of DQN, Rainbow, and the C51 variant of Rainbow 
+within their framework [Dopamine](https://github.com/google/dopamine). Using the Gym
+wrapper, we can run Unity environments using Dopamine. 
+
+First, after installing the Gym wrapper, clone the Dopamine repository. 
+
+```
+git clone https://github.com/google/dopamine
+```
+
+Then, follow the install instructions as specified on 
+[Dopamine's homepage](https://github.com/google/dopamine). Don't install as a package yet,
+as you will need to make some changes to the files. 
+
+### Adapting Dopamine's Scripts
+
+First, open `dopamine/atari/run_experiment.py`. Alternatively, copy the entire `atari` 
+folder, and name it something else (e.g. `unity`). If you choose the copy approach,
+be sure to change the package names in the import statements in `train.py` to your new
+directory.
+
+Within `run_experiment.py`, we will need to make changes to which environment is 
+instantiated, just as in the Baselines example. At the top of the file, insert 
+
+```python
+from gym_unity.envs import UnityEnv
+```
+
+to import the Gym Wrapper. Navigate to the `create_atari_environment` method 
+in the same file, and switch to instantiating a Unity environment by replacing
+the method with the following code. 
+
+```python
+    game_version = 'v0' if sticky_actions else 'v4'
+    full_game_name = '{}NoFrameskip-{}'.format(game_name, game_version)
+    env = UnityEnv('./envs/GridWorld', 0, use_visual=True, uint8_visual=True)
+    return env
+```
+
+`./envs/GridWorld` is the path to your built Unity executable. For more information on 
+building Unity environments, see [here](../docs/Learning-Environment-Executable.md), and note 
+the Limitations section below. 
+
+Note that we are not using the preprocessor from Dopamine, 
+as it uses many Atari-specific calls. Furthermore, frame-skipping can be done from within Unity, 
+rather than on the Python side. 
+
+### Limitations
+
+Since Dopamine is designed around variants of DQN, it is only compatible
+with discrete action spaces, and specifically the Discrete Gym space. For environments
+that use branched discrete action spaces (e.g. 
+[VisualBanana](../docs/Learning-Environment-Examples.md)), you can enable the 
+`flatten_branched` parameter in `UnityEnv`, which treats each combination of branched 
+actions as separate actions.
+
+Furthermore, when building your environments, ensure that your
+[Learning Brain](../docs/Learning-Environment-Design-Brains.md) is using visual
+observations with greyscale enabled, and that the dimensions of the visual observations
+is 84 by 84 (matches the parameter found in `dqn_agent.py` and `rainbow_agent.py`).
+Dopamine's agents currently do not automatically adapt to the observation 
```

Comment:
When comparing GridWorld to OpenAI's baselines and PPO, make sure we use the same grayscale build.